xml/System.Web.UI.WebControls/LinqDataSourceView.xml

```diff
@@ -2600,10 +2835,12 @@
         <Parameter Name="value" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The name of the parameter to validate.</param>
+        <param name="value">The value of the parameter to validate.</param>
+        <summary>Validates that the OrderBy parameter value is a valid field name for the automatically generated OrderBy clause.</summary>
```

Comment:
I think "OrderBy" needs code brackets to prevent translation.

Reply:
Ditto